### PR TITLE
refactor: use API token instead of legacy API key

### PR DESCRIPTION
**Pull request by twsansbury**
Some products provides usage examples for the legacy API keys authorization scheme.  Where possible, the documentation and usage examples should direct users to use the recommended API tokens authorization scheme.

### CODE REVIEW

**Review by bkrebsbach — APPROVED** [review]
Stream changes LGTM

**Comment by twsansbury** [comment]
@bkrebsbach, the stream documentation was updated in #149 to start using API tokens.  The GraphQL usage example uses an API token:

https://github.com/cloudflare/cloudflare-docs/blob/9a120a4ff56308093c2818ec00c58f4179a8d876/products/stream/src/content/getting-analytics/fetching-bulk-analytics.md#L78-L82

However, the document explicitly states API tokens cannot be used:

https://github.com/cloudflare/cloudflare-docs/blob/9a120a4ff56308093c2818ec00c58f4179a8d876/products/stream/src/content/getting-analytics/fetching-bulk-analytics.md#L234-L235

These seem to be in conflict with one another.  If I knew which was correct, I would be glad to fix the documentation as part of this pull request.

**Comment by bkrebsbach** [comment]
@Everlag / @renandincer - do we not support API tokens? See @twsansbury's comments:
> @bkrebsbach, the stream documentation was updated in #149 to start using API tokens. The GraphQL usage example uses an API token:
> 
> https://github.com/cloudflare/cloudflare-docs/blob/9a120a4ff56308093c2818ec00c58f4179a8d876/products/stream/src/content/getting-analytics/fetching-bulk-analytics.md#L78-L82
> 
> However, the document explicitly states API tokens cannot be used:
> 
> https://github.com/cloudflare/cloudflare-docs/blob/9a120a4ff56308093c2818ec00c58f4179a8d876/products/stream/src/content/getting-analytics/fetching-bulk-analytics.md#L234-L235
> 
> These seem to be in conflict with one another. If I knew which was correct, I would be glad to fix the documentation as part of this pull request.

**Comment by bkrebsbach** [comment]
@twsansbury The limitation requiring API keys is no longer valid, I've confirmed that API tokens work for Stream.

**Review by tlozoot — APPROVED** [review]
Thanks, LGTM. fyi @crwaters16 as well

File: products/analytics/src/content/graphql-api/getting-started/authentication/graphql-client-headers.md [hunk 1]
```diff
@@ -12,13 +12,13 @@ order: 40
...
-1. To configure authentication, click **Add Header**. You can use Cloudflare Analytics API token authentication (recommended) or Cloudflare API key authentication.
+1. To configure authentication, click **Add Header**. You can use Cloudflare API token authentication (recommended) or Cloudflare API key authentication (legacy).
    * **Token authentication**:
    Enter **Authorization** in the _Header Name_ field, and enter **Bearer {your-analytics-token}**  in the _Header value_ field, then click **Save**.
    ![HTTP Headers](../../../static/images/GraphiQL-edit-http-headers-token.png)
-   * **Key authentication**:
-   Enter **X-AUTH-KEY** in the _Header Name_ field, and paste your Global API Key in the _Header value_ field, then click **Save**.
+   * **Key authentication (legacy)**:
+      1. Enter **X-AUTH-EMAIL** in the _Header name_ field and your email address registered with Cloudflare in the _Header value_ field.
+      1. Enter **X-AUTH-KEY** in the _Header Name_ field, and paste your Global API Key in the _Header value_ field, then click **Save**.
```
Comment (crwaters16) [inline products/analytics/src/content/graphql-api/getting-started/authentication/graphql-client-headers.md]: nit: delete comma after first "field"

File: products/analytics/src/content/graphql-api/tutorials/build-your-own-analytics/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ In this example, we're going to see how to use the GraphQL Analytics API to buil
 
 ![GraphQL recipe](../static/graphQL-recipe-cacheVisual.gif)
 
-The following code will build a page with all the requirement to fetch from GraphQL and plot the cached and uncached bandwidth for the given zone. You'll just need to enter your email address, API token, and your zone ID, and then push the `Fetch analytics` button.
+The following code will build a page with all the requirement to fetch from GraphQL and plot the cached and uncached bandwidth for the given zone. You'll just need to enter your API token and your zone ID, and then push the `Fetch analytics` button.
```
Comment (crwaters16) [inline products/analytics/src/content/graphql-api/tutorials/build-your-own-analytics/index.md]: Small edits:
-Requirements should be plural
-Delete "just"
-Delete the "and" after Zone ID - "...toke and your zone ID, then push the..."

Comment (twsansbury) [reply products/analytics/src/content/graphql-api/tutorials/build-your-own-analytics/index.md]: Done.

File: products/analytics/src/content/graphql-api/getting-started/authentication/graphql-client-headers.md (hunk 1 above)
Comment (twsansbury) [reply products/analytics/src/content/graphql-api/getting-started/authentication/graphql-client-headers.md]: Done.

**Review by soheiokamoto — APPROVED** [review]

**Comment by twsansbury** [comment]
Given the number of code owners, should I split this into different merge requests?  I am worried I will continuously need to rebase and resolve merge conflicts while waiting for approval from other code owners.

File: products/ssl/src/content/ssl-for-saas/certificate-validation-methods.md [hunk 3]
```diff
@@ -264,4 +264,4 @@ $ curl -sXPATCH "https://api.cloudflare.com/client/v4/zones/{zone_id}/custom_hos
 
 Next, you will need to add the CNAME record that is provided in the results, ie `name: _ca3-fbc2086e83a647d4822fefa68f26fc55.cname.example.com` and `cname_target:dcv.digicert.com`, at the Authoritative DNS provider for the hostname. This CNAME record information can also be located in the Custom Hostname section of the SSL dashboard.
 
-The certificate should validate relatively soon after its added. If you’d like to check immediately simply send a `PATCH` with the same payload.
\ No newline at end of file
+The certificate should validate relatively soon after its added. If you’d like to check immediately simply send a `PATCH` with the same payload.
```
Comment (crwaters16) [inline products/ssl/src/content/ssl-for-saas/certificate-validation-methods.md:267]: Should be "it is" or "it's"

**Comment by KimJ15** [comment]
This PR was opened before the developer doc site was moved to Cloudflare Pages. There are many conflicting branches. I'm going to close it now, but feel free to open a new issue or ask any questions.

**Pull request closed without merging**